index.rst

```diff
@@ -66,7 +66,7 @@ Configuration
    doc/kinematics_configuration/kinematics_configuration_tutorial
    doc/custom_constraint_samplers/custom_constraint_samplers_tutorial
    doc/ompl_interface/ompl_interface_tutorial
-   doc/chomp_interface/chomp_interface_tutorial
+   doc/chomp_planner/chomp_planner_tutorial
```

Comment:
@mlautman lets no do any more changes in the PR, this one is fine, but in the future I think you should *avoid* asking people to rename folders and filenames in these tutorials, as that breaks hyperlinks on a live website which should be generally avoided

Reply:
I agree about merging. With regard to filenames and directories, I think that it's important that we maintain the standards in the contributing guidelines. That said, this tutorial could have kept the name `chomp_interface_tutorial` and I wouldn't have said anything. In the future, I'll probably just suggest that the name of the tutorial stays the same rather than suggest that the file and dir name are also changed to match.